cli/pkg/kctrl/cmd/package/repository/add_or_update.go

```diff
@@ -141,6 +145,24 @@ func (o *AddOrUpdateOptions) Run(args []string) error {
 		return err
 	}
 
+	if o.CreateNamespace {
+		coreClient, err := o.depsFactory.CoreClient()
+		if err != nil {
+			return err
+		}
+
+		namespace := &v1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: o.NamespaceFlags.Name,
+			},
+		}
+
```

Comment:
Let's add a log here saying that we are going to create a namespace. Something like `Creating namespace 'asdf'`.
Let's use `o.statusUI.PrintMessagef()` for this. (Adds a nice little timestamp in front of the message)

I think the message we are printing when namespace already exists should use `statusUI` as well 🤔

Reply:
"here" being right before we make the call to create it

Reply:
Done. I have used `statusUI` for both messages.

Reply:
@100mik @ThomasVitale Instead of saying `Creating namespace xyz` and `Namespace xyz already exists` every time (thinking about updated scenarios), should we say `Created namespace xyz` once when we create the namespace?

(No strong opinions)

Reply:
I like that! I have updated the code to only print a message if the namespace gets created.